x-pack/plugins/alerting/common/maintenance_window.ts

```diff
@@ -63,3 +63,60 @@ export const MAINTENANCE_WINDOW_API_PRIVILEGES = {
   READ_MAINTENANCE_WINDOW: 'read-maintenance-window',
   WRITE_MAINTENANCE_WINDOW: 'write-maintenance-window',
 };
+
+export const MAINTENANCE_WINDOWS_APP_ID = 'maintenanceWindows';
+export const MANAGEMENT_APP_ID = 'management';
+
+export const MAINTENANCE_WINDOW_PATHS = {
+  alerting: {
+    maintenanceWindows: `/${MAINTENANCE_WINDOWS_APP_ID}`,
+    maintenanceWindowsCreate: '/create',
+    maintenanceWindowsEdit: '/edit/:maintenanceWindowId',
+  },
+};
+
+export const MAINTENANCE_WINDOW_DEEP_LINK_IDS = {
+  maintenanceWindows: MAINTENANCE_WINDOWS_APP_ID,
+  maintenanceWindowsCreate: 'create',
+  maintenanceWindowsEdit: 'edit',
+};
+
+export type MaintenanceWindowDeepLinkIds =
+  typeof MAINTENANCE_WINDOW_DEEP_LINK_IDS[keyof typeof MAINTENANCE_WINDOW_DEEP_LINK_IDS];
+
+export const TABLE_STATUS_RUNNING = i18n.translate(
```

Comment:
do we really need to export all these translation? I have a weird feeling to have these translations in common

Reply:
I can move it but this is definitely more convenient as we need to share statuses between plugins

Reply:
WIll end up with duplication